openapi/components/schemas/GatewayAccountConfig/MobilePay.yaml

```
@@ -0,0 +1,29 @@
+description: MobilePay gateway configuration.
+allOf:
+  - $ref: ../GatewayAccount.yaml
+  - type: object
+    required:
+      - credentials
+    properties:
+      credentials:
+        type: object
+        properties:
+          providerId:
+            type: string
+            description: MobilePay provider ID.
+          merchantVat:
+            type: string
+            description: MobilePay merchant VAT.
+          clientId:
+            type: string
+            description: MobilePay client ID.
```

nitpick: where possible, avoid using three or more nouns in a row 
(it can be difficult to read)
```suggestion
            description: ID of the MobilePay provider.
          merchantVat:
            type: string
            description: VAT number of the MobilePay merchant.
          clientId:
            type: string
            description: ID of the MobilePay client.
```